shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -132,26 +132,28 @@ class JSBackend {
       JSField(translateTerm(receiver), fieldName.name)
     // Turn let into an IIFE.
     case Let(true, Var(name), Lam(args, body), expr) =>
-      val letScope = scope.derive("Let", name :: Nil)
+      val letScope = scope.derive("Let")
+      val runtimeParam = letScope.declareParameter(name)
```

Comment:
Why do you call this `Param`? It's not a parameter; it's the function's name!

Reply:
Oh ok it's because it's the parameter in the following `JSImmEvalFn`. Still, the name is quite confusing.

Reply:
What about `runtimeName`?

Reply:
Sure.